sample/src_polymer/scripts/common/utils.js

```diff
@@ -0,0 +1,13 @@
+var RSUtils = {
```

Comment:
Maybe we'd better place both `spMath.js` and `utils.js` to `src_polymer/common`, which is parallel to `src_polymer/libs` as before?